Getting-Started.md

```diff
@@ -155,6 +155,28 @@ LimePayWeb.init(limeToken, limePayConfig).catch((err) => {
 
 The `.init()` has 2 parameters. The first parameter is the `x-lime-token` that is received when you create your payment (described in [6. Create Payment](#create-payment)) and the second one is the `config` object that we described in step `2)`
 
+### 7.1 Utils
+* calculateVAT (**cardHoldeVATData**)   
+When you initialize LimePayWeb you can get the vat/tax data (if any) for a payment.   
```

Comment:
I think that it would be better if we write it:
`Once you initialize ... ` because if you type `when you initialize` it oposes that in the moment when you are initializing the LimePayWeb, you can calculate the VAT

Reply:
You can add additional information that the whenever our clients are creating Payments (from the backend) they are creating the payment without VAT/TAX, meaning that if they want to show the TOTAL that their users will be charged, they should calculate the TAX/VAT and show it as-well, so that their users won't get confused.